### ISSUE TITLE
[MU4 Issue] Cue size option for noteheads is missing

### ISSUE THREAD

**Issue by oktophonie** [musescore/MuseScore#10003]
In MU3 we could set the _chord_ to 'Small',  but could also set individual _noteheads_ to 'Small':

![image](https://user-images.githubusercontent.com/20604032/145041607-a9e9f638-a072-469b-af95-7ee160d1d776.png)

This latter option seems to have vanished in MU4; we can still make the entire chord small (now called 'Cue size') but can't control noteheads individually:

![image](https://user-images.githubusercontent.com/20604032/145041854-8b75acf6-e596-4bcc-90a3-3643ae0f98eb.png)

Perhaps 'Cue size' should also appear in the 'Head' section, or the 'Cue size' option in 'General' could have some clever behaviour. @Tantacrul thoughts?

**Comment by Tantacrul** [comment]
My original intention with the new design was that a user could use the 'Cue size' option to alter an individual notehead _and_ also a whole chord (if they selected the whole chord).

This would be my strong preference from a UX point of view. If for some reason, this is very hard to achieve, then perhaps we could have an individual setting under the 'Head' tab. I don't really like that second solution though.

@Eism @RomanPudashkin @vpereverzev ?

**Comment by oktophonie** [comment]
One important(ish) point is that the MU3 behaviour under 'Note' just changes the size of the notehead but nothing else (doesn't change the stem thickness, flag size, dots, etc, or the alignment). If we have only one 'Cue size' option in MU4, in the case of a single note 'chord, would we be able to distinguish between changing just the size of the notehead vs all the components of the note?

**Comment by Tantacrul** [comment]
You might be right. I thought the larger 'group' selection might handle that case but there are too many problems with it. I won't go into it here.

I think we'll need a design tweak, with a tickbox specifically in the 'Head' section. @bkunda: we'll need a new design!

**Comment by shoogle** [comment]
In MS3, users didn't always understand the difference between making the entire chord small vs. making just the notehead small, yet both options are needed.

I'd think "Cue size" should affect the entire chord since that is the common case. Then there should be a "Small notehead" checkbox to right of "Hide notehead" to handle the (rare) occasion where you actually need to make the notehead itself small.

The next question is what should happen if both the chord and the notehead are small? In MS3, this results in the notehead becoming even smaller still, which seems a bit odd but is probably the safest option. The alternative would be to disable the "Small notehead" option if the chord is made cue sized.

**Comment by Tantacrul** [comment]
@shoogle: agree. My vote would be that if 'cue size' and 'small note head' are both activated, the notes do not get doubly small. We could potentially make the 'small note head' check box disabled if 'cue size' is switched on. Don't think it massively matters though.

**Comment by oktophonie** [comment]
You do need 'double small' notes - if you have grace notes within the context of a cue!

**Comment by Tantacrul** [comment]
One less problem to solve then

**Comment by Tantacrul** [comment]
This was added by someone at some point. :)

You can make individual notes cues. And you can make the whole chord a cue too.

<img width="720" alt="image" src="https://user-images.githubusercontent.com/21022311/170733132-8e60d4af-5503-4c94-8073-4f0ea16c89bf.png">

Closing as fixed.
